config/initializers/ab_tests.rb

```diff
@@ -0,0 +1,21 @@
+require 'ab_test_bucket'
+
+module AbTests
+  NATIVE_CAMERA = AbTestBucket.new(
+    experiment_name: 'Native Camera Only',
+    buckets: {
+      native_camera_only: IdentityConfig.store.idv_native_camera_a_b_testing_enabled ?
+        IdentityConfig.store.idv_native_camera_a_b_testing_percent :
+        nil,
```

Comment:
In my opinion, we shouldn't need to have *both* an "enabled" feature flag as well as a percent feature flag... we just set the percent to zero (or -1) and boom, everything goes to the default.

I figured that change would require changes to the workflow of whoever manages these flags, so I opted not to change them here

Reply:
Agreed, that would work for cases like this with only one alternative (which is likely to be the norm).

Reply:
Well, we also may want to suppress event logging if the test is "disabled"? That doesn't happen in AbTestBucket (though maybe it should?)

Reply:
I think that we should be logging which buckets get assigned in the particular events, because with Cloudwatch today, we can't join in two separate events easily, so my vote is that AbTestBucket should not do logging itself